streaming/base/converters/__init__.py

```diff
@@ -0,0 +1,8 @@
+# Copyright 2023 MosaicML Streaming authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Utility function for converting spark dataframe to MDS dataset."""
+
+from streaming.base.converters.dataframe_to_mds import MAPPING_SPARK_TO_MDS, dataframeToMDS
+
+__all__ = ['dataframeToMDS', 'MAPPING_SPARK_TO_MDS']
```

Comment:
Does `MAPPING_SPARK_TO_MDS` getting use anywhere apart from `dataframe_to_mds.py` ? If not, then we don't need to include here !!

Reply:
It is not used in the code, but I think it is useful for users to import it and inspect the mapping from a notebook?